calculator.js

```diff
@@ -1,28 +1,121 @@
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+// const exampleAdditionInput = {
+//   num1: 3,
+//   num2: 5,
+//   operation: 'add',
+// }
+// // '3 + 5 = 8'
+
+
+const addNums = function(a, b) {
+  return `${a} + ${b} = ${a + b}`;
+}
+
+const subtractNums = function(a, b) {
+  return `${a} - ${b} = ${(a - b)}`;
+}
+
+const multiplyNums = function(a, b) {
+  return `${a} * ${b} = ${(a * b)}`;
+}
+
+const divideNums = function(a, b) {
+  return `${a} / ${b} = ${(a / b)}`;
+}
+
+const notNumber = function(a, b) {
+  
+  if (isNaN(a)) { 
+    return `${a} is not a number.`
+  } else {
+    return `${b} is not a number.`
+  }
+  
 }
-// '3 + 5 = 8'
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
-
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  
+  const a = Number(promptInput.num1);
+  const b = Number(promptInput.num2);
```

Comment:
consider using variable names that are more reflective of the original key names